src/main/resources/assets/ingameinfo/lang/de_DE.lang

```diff
@@ -1,45 +1,284 @@
+ingameinfoxml.tag.chestplateuniquename.desc=Eindeutiger Rüstungsname.
+ingameinfoxml.tag.chestplatedamage.desc=Rüstungsschaden.
+ingameinfoxml.tag.chestplatemaxdamage.desc=Maximaler Rüstungsschaden.
+ingameinfoxml.tag.chestplatedamageleft.desc=Restlicher Rüstungsschaden.
+ingameinfoxml.tag.chestplatequantity.desc=Anzahl der Rüstungen im Inventar.
+ingameinfoxml.tag.chestplateicon.desc=Rüstungsicon.
+ingameinfoxml.tag.chestplatelargeicon.desc=Großer Rüstungsicon.
+ingameinfoxml.tag.leggingsname.desc=Beinschienenname.
+ingameinfoxml.tag.leggingsuniquename.desc=Eindeutiger Beinschienenname.
+ingameinfoxml.tag.leggingsdamage.desc=Schaden der Beinschienen.
+ingameinfoxml.tag.leggingsmaxdamage.desc=Maximaler Schaden der Beinschienen.
+ingameinfoxml.tag.leggingsdamageleft.desc=Restschaden der Beinschienen.
+ingameinfoxml.tag.leggingsquantity.desc=Anzahl der Beinschienen im Inventar.
+ingameinfoxml.tag.leggingsicon.desc=Beinschienenicon.
+ingameinfoxml.tag.leggingslargeicon.desc=Großer Beinschienenicon.
+ingameinfoxml.tag.bootsname.desc=Stiefelname.
+ingameinfoxml.tag.bootsuniquename.desc=Eindeutiger Stiefelname.
+ingameinfoxml.tag.bootsdamage.desc=Stiefelschaden.
+ingameinfoxml.tag.bootsmaxdamage.desc=Maximaler Stiefelschaden.
+ingameinfoxml.tag.bootsdamageleft.desc=Restlicher Stiefelschaden.
+ingameinfoxml.tag.bootsquantity.desc=Anzahl der Stiefel im Inventar.
+ingameinfoxml.tag.bootsicon.desc=Stiefelicon.
+ingameinfoxml.tag.bootslargeicon.desc=Großer Stiefelicon.
 
 # gui - tag list - player general
+ingameinfoxml.tag.light.desc=Lichtpegel an der aktuellen Position.
+ingameinfoxml.tag.lightfeet.desc=Lichtpegel an der aktuellen Position (Sichtebene).
+ingameinfoxml.tag.lightnosun.desc=Lichtpegel an der aktuellen Position ohne Sonnenlicht.
+ingameinfoxml.tag.lightnosunfeet.desc=Lichtpegel an der aktuellen Position ohne Sonnenlicht (Sichtebene).
+ingameinfoxml.tag.score.desc=Spieler Score.
+ingameinfoxml.tag.gamemode.desc=Spieler Gamemode Name.
+ingameinfoxml.tag.gamemodeid.desc=Spieler Gamemode ID.
+ingameinfoxml.tag.health.desc=Aktuelle Lebenspunkte des Spielers.
+ingameinfoxml.tag.maxhealth.desc=Maximale Lebenspunkte des Spielers.
+ingameinfoxml.tag.armor.desc=Rüstungspunkte des Spielers.
+ingameinfoxml.tag.foodlevel.desc=Essenspunkte des Spielers.
+ingameinfoxml.tag.maxfoodlevel.desc=Maximale Essenspunkte des Spielers.
+ingameinfoxml.tag.saturation.desc=Spieler Sättigung.
+ingameinfoxml.tag.exhaustion.desc=Spieler Hunger.
+ingameinfoxml.tag.airticks.desc=Spieler Luft.
+ingameinfoxml.tag.maxairticks.desc=Maximale Spieler Luft.
+ingameinfoxml.tag.playerlevel.desc=Spieler Level.
+ingameinfoxml.tag.xpthislevel.desc=Aktuelle Spieler-Erfahrung.
+ingameinfoxml.tag.xpuntilnext.desc=Spieler-Erfahrung bis zum Level-Aufstieg.
+ingameinfoxml.tag.xpcap.desc=Gesamte Spieler-Erfahrung für das aktuelle Spieler-Level.
+ingameinfoxml.tag.username.desc=Benutzername des Spielers.
+ingameinfoxml.tag.inwater.desc=§eJa§r wenn der Spieler taucht, sonst §eNein§r.
```

Comment:
true/false shouldn't be translated. It's the literal value that's returned by the tag.